script/dancer2

```diff
@@ -94,93 +81,26 @@ how to convert a Dancer (1) based application to Dancer2.
 
 =item gen                   : create new Dancer2 application
 
-=item commands              : list the application's commands
-
-=item help                  : display a command's help screen
-
 =item version               : display version
 
 =back
 
 To get detailed description of each individual command run:
-    dancer2 help <command>
-
-The lastest list of available commands can be dispayed by:
-    dancer2 commands
-
-=head1 COMMAND 'gen'
-
-Helper script for providing a bootstrapping method to quickly and easily create
-the framework for a new Dancer2 application.
-
-=head3 OPTIONS
-
-        -a --application     the name of your application
-        -p --path            the path where to create your application
-                             (current directory if not specified)
-        -o --overwrite       overwrite existing files
-        -x --no-check        don't check for the latest version of Dancer2
-                             (checking version implies internet connection)
-        -s --skel            skeleton directory
-
-=head3 EXAMPLE
-
-Here is an application created with dancer2:
-
-    $ dancer2 gen -a MyWeb::App
-    + MyWeb-App
-    + MyWeb-App/bin
-    + MyWeb-App/bin/app.psgi
-    + MyWeb-App/config.yml
-    + MyWeb-App/environments
-    + MyWeb-App/environments/development.yml
-    + MyWeb-App/environments/production.yml
-    + MyWeb-App/views
-    + MyWeb-App/views/index.tt
-    + MyWeb-App/views/layouts
-    + MyWeb-App/views/layouts/main.tt
-    + MyWeb-App/MANIFEST.SKIP
-    + MyWeb-App/lib
-    + MyWeb-App/lib/MyWeb
-    + MyWeb-App/lib/MyWeb/App.pm
-    + MyWeb-App/public
-    + MyWeb-App/public/css
-    + MyWeb-App/public/css/style.css
-    + MyWeb-App/public/css/error.css
-    + MyWeb-App/public/images
-    + MyWeb-App/public/500.html
-    + MyWeb-App/public/404.html
-    + MyWeb-App/public/dispatch.fcgi
-    + MyWeb-App/public/dispatch.cgi
-    + MyWeb-App/public/javascripts
-    + MyWeb-App/public/javascripts/jquery.js
-    + MyWeb-App/t
-    + MyWeb-App/t/002_index_route.t
-    + MyWeb-App/t/001_base.t
-    + MyWeb-App/Makefile.PL
-
-The application is ready to serve:
-
-    $ cd MyWeb-App
-    $ plackup bin/app.psgi
-    >> Listening on 127.0.0.1:3000
-    == Entering the development dance floor ...
+    dancer2 <command> --help
 
-=head1 AUTHOR
+The lastest list of available commands can be displayed by:
+    dancer2 
 
```

Comment:
i'm torn here; on the one hand, I think the list of commands should be documented in a way you can easily see without having to run the code to find out.

On the other hand, that means we have to remember to keep that documentation in sync with the actual CLI app implementation... hmm.

Reply:
I wonder if we could get cute and auto-generate the doco for it during the build process?  (If we think that would be good, that's probably one to raise as a new ticket to implement separately if anyone chooses to do so, and not block this PR with...)

Reply:
The two subcommands we have are documented when you run `perldoc dancer2`. I think having the option to see the subcommands when running the command without args is nice.

As far as auto-generating the docs for this script - I would LOVE this. I will add it to my list of possible documentation fixes.